deployments/roles/install_noisepage_dependencies/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Do we still need this. I'm guessing this was for the SQLancer experiments

Reply:
yeah you are right. i had it removed.